build/Packages.props

```diff
@@ -60,7 +60,7 @@
     <PackageReference Update="Nuget.ProjectModel" Version="$(NuGetPackageVersion)" />
     <PackageReference Update="Nuget.Versioning" Version="$(NuGetPackageVersion)" />
 
-    <PackageReference Update="OmniSharp.Extensions.LanguageServer" Version="0.11.3" />
+    <PackageReference Update="OmniSharp.Extensions.LanguageServer" Version="0.11.4-alpha0011" />
```

Comment:
Needs stable release.

Reply:
What is this package?

Reply:
@rchande it's the LSP bits from [here](https://github.com/OmniSharp/csharp-language-server-protocol)